jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/PlatformsParameters.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.gradle;
+
+import com.google.cloud.tools.jib.plugins.common.RawConfiguration.PlatformsConfiguration;
+import javax.annotation.Nullable;
+
+/** Configuration of a platform. */
+public class PlatformsParameters implements PlatformsConfiguration {
+  @Nullable String os;
+  @Nullable String architecture;
+
+  @Override
+  public java.util.Optional<String> getOs() {
+    return java.util.Optional.ofNullable(this.os);
+  }
```

Comment:
There's no other class with the same name `Optional` used in this file, so you don't have to fully qualify the class. Changing the code like below will make your IDE automatically add `import java.util.Optional` at the top.

```suggestion
  @Override
  public Optional<String> getOs() {
    return Optional.ofNullable(this.os);
  }
```